src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/monitor/monitor.component.html

```diff
@@ -0,0 +1,115 @@
+<!-- Page Header -->
+<h1 class="name">
+  Monitors
+</h1>
+
+<!-- Main content -->
+<div class="row">
```

Comment:
Should be replace with the data table component.

Reply:
I'm now using the cdTable on both quorum tables.